src/components/VContentSwitcher/VSearchTypeButton.vue

```diff
@@ -22,20 +22,22 @@
   </VButton>
 </template>
 <script>
-import { ALL_MEDIA, supportedSearchTypes } from '~/constants/media'
 import {
   computed,
   defineComponent,
   inject,
   useContext,
 } from '@nuxtjs/composition-api'
+
+import { ALL_MEDIA, supportedSearchTypes } from '~/constants/media'
 import useSearchType from '~/composables/use-search-type'
 import { isMinScreen } from '~/composables/use-media-query'
 
-import caretDownIcon from '~/assets/icons/caret-down.svg'
+import VIcon from '~/components/VIcon/VIcon.vue'
```

Comment:
It only reorders the imports, and does not add empty lines between groups, right? The blank line between two components # 37 could be removed, too.